tests/test_notifications.py

```
@@ -851,3 +852,64 @@ def test_issue_reopened_without_assignee() -> None:
+
+def test_pr_opened_channel_notification_posts_to_mapped_channel() -> None:
+    storage = MockStorage()
+    storage.verified_mappings = [{"discord_user_id": "999", "github_user": "alice"}]
+    discord_writer = MockDiscordWriter()
+    config = NotificationConfig(enabled=True, pr_opened=True)
+    policy = MutationPolicy(mode=RunMode.ACTIVE, github_write_allowed=True, discord_write_allowed=True)
+    event = ContributionEvent(
+        github_user="alice",
+        event_type="pr_opened",
+        repo="Gitcord-GithubDiscordBot",
+        created_at=datetime.now(timezone.utc),
```

Ruff (0.15.20)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)